src/Mvc/test/Mvc.FunctionalTests/ApiExplorerTest.cs

```diff
@@ -607,7 +607,7 @@ public async Task ExplicitResponseTypeDecoration_SuppressesDefaultStatus_AlsoHon
             // Arrange
             var type1 = typeof(ApiExplorerWebSite.Product).FullName;
             var type2 = typeof(SerializableError).FullName;
-            var expectedMediaTypes = new[] { "text/xml" };
+            var expectedMediaTypes = new[] { "application/xml", "text/xml", "application/json", "text/json" };
```

Comment:
This looks wrong - the action says that it only returns xml:

https://github.com/dotnet/aspnetcore/blob/8b30d862de6c9146f466061d51aa3f1414ee2337/src/Mvc/test/WebSites/ApiExplorerWebSite/Controllers/ApiExplorerResponseTypeWithAttributeController.cs#L66-L69

We can't change this.